Claude-cowork-best-practices.html

```
@@ -0,0 +1,2002 @@
+          <h3>判断フロー</h3>
+          <div class="diagram-wrap">
+            <pre class="mermaid" data-src="d-decision"></pre>
+          </div>
+
+          <h3>Cowork向きタスクの「5つの材料」</h3>
+          <p class="muted">
+            すべてを満たす必要はありませんが、当てはまる項目が多いほどCowork向きです。
+          </p>
+          <ul class="checklist">
+            <li>
+              <input type="checkbox" disabled /><span
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)